recipes/plumed/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "plumed" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plumed/plumed2/archive/v{{ version }}.tar.gz
+  sha256: de309980dcfd6f6e0e70e138856f4bd9eb4d8a513906a5e6389f18a5af7f2eba
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+# Most programs should link libplumedWrapper.a.
+# They will be forward/backward compatible with all plumed 2 releases.
+    - {{ pin_subpackage('plumed', min_pin='x', max_pin='x') }}
```

Comment:
If most people are statically linking then it doesn't need to be pinned as it isn't needed at runtime?

Reply:
You are right. libplumedWrapper.a is using dlopen to load libplumedKernel.so, only when plumed functionalities are requested. Thus, no need for pinning.

Reply:
Sorry I missed the word wrapper. Would users ever not want the "plumed functionalities"?

(Potential library loading errors are worse than extra dependencies. They're especially hard to debug if a different installation is found from the host)

Reply:
The way it works is that molecular dynamics codes using plumed library either contain the code that we distribute in `libplumedWrapper.a` (in which case they do not need plumed at build time) or link statically to `libplumedWrapper.a` (in which case they do need plumed at build time).

At run time, when the user chooses to use one of the methods provided by the plumed library, dlopen is called. By default it will look in `condaprefix/lib/libplumedKernel.so`, but this can be customized with `export PLUMED_KERNEL=/path/to/libplumedKernel.so` so that one can provide his own plumed compilation. The interface is pure C, so there is no problem with c++ library versions.

Thus, if plumed is not available at runtime, the user will receive an error asking to set `PLUMED_KERNEL` env var. If you prefer I can pin again the plumed package, with loose constraints on versions as I did first.

Reply:
I think it's okay then and you can always change it and bump the build number if you run into issues.

LGTM 👍